docs/process/testing.md

```diff
@@ -0,0 +1,233 @@
+### Download Desktop
+  - [ ] User can download latest (Mac & Windows) Desktop from https://desktop.github.com/    
+    - [ ] Mac: https://central.github.com/mac/latest 
+      - [ ] Windows: https://github-windows.s3.amazonaws.com/GitHubSetup.exe
+  - [ ] Release notes page is up-to-date
+  - [ ] Help page is accessible
+
+### Welcome Flow
+  - [ ] Create your account (/join?source=github-desktop)
+    - [ ] User is not automatically logged into Desktop post creation
+  - [ ] Sign in to Github.com 
```

Comment:
clarified it is a link, but no hyperlink to dotcom as that is not part of welcome flow